tests/src/rules/order.js

```diff
@@ -1285,25 +1299,8 @@ ruleTester.run('order', rule, {
           import blah = require('./blah');`,
         parser,
         errors: [{
-          message: '`./blah` import should occur before import of `console.log`',
-        }],
-      }),
-      // Alphabetization of object-imports
-      test({
-        code: `
-          import log = console.log;
-          import debug = console.debug;`,
-        parser,
-        errors: [{
-          message: '`console.debug` import should occur before import of `console.log`',
+          message: '`./blah` import should occur before import of ``',
```

Comment:
this isn't a very clear error message; is there a better way we can do this?

Reply:
I gave it a go.
Do you think, this change is fine?